app/components/Channels/messages.js

```diff
@@ -43,7 +43,10 @@ export default defineMessages({
   close_button: 'Close Channel',
   force_close_button: 'Force Close Channel',
   unknown: 'Unknown',
+  funding_date_label: 'Channel opening date',
+  funding_date_description: 'Channel funding transaction date and time',
```

Comment:
Description should end with a period for consistency with all the other row descriptions.

Reply:
updated